tsconfig.json

```diff
@@ -12,6 +12,5 @@
     "outDir": "lib",
     "sourceMap": true,
     "moduleResolution": "nodenext"
-  },
```

Comment:
There is no need to declare the included files since we now rely on an entry point declared on each project (aka `tsconfig.json`).